tools/serverpod_cli/lib/src/util/locate_modules.dart

```diff
@@ -79,6 +96,27 @@ class _ModuleGeneratorConfigLite {
   }
 }
 
+List<String> findAllMigrationVersionsSync({
+  required Directory directory,
+  required String moduleName,
+}) {
+  try {
+    var migrationRoot = Directory(path.join(
+      MigrationConstants.migrationsBaseDirectory(directory).path,
+      moduleName,
+    ));
+
+    var migrationsDir = migrationRoot.listSync().whereType<Directory>();
+
+    var migrationVersions =
```

Comment:
Suggestion: Sort this list.

This list needs to be sorted in order for us to use it.